### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by AlexRosito67**
Removal of
https://github.com/AlexRosito67/FSSD
Because of several bugs

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @AlexRosito67. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2023/07/26 10:47:16 Loaded 6042 libraries from DB
Removing FSSD
warning: Library Git clone folder /home/ubuntu/workspace/gitclones/github.com/AlexRosito67/FSSD not present
Success!
```

**Pull request merged**
